### ISSUE TITLE
New eqiupement slot, DOTA attack, bonuses on tiles, spells 

### ISSUE THREAD

**Issue by bylinozer** [VladimirMakeev/D2ModdingToolset#143]
**Proposal:** 
It would be nice to unlock new equipment slot which theoretically was supposed to be added to game. Armor type slot.

**Proposal2:**
Currently I think that changing or adding new custom dota type attack does`t work correctly, e.g when unit have it already in Gunits (2nd attack frostbite), then changing it for different one doesn't work for GDynUpgr, it only include attack value from Gattacks.

**Proposal3:**
Bonuses(attack, def, int etc) on different tile.terrain or ground, which that includes terrain changes from end of turn, rods etc. 

**Proposal4:**
In setting.lua adding new function which would deffine how many times per day we could use given spell and if possible include it  together with Spell Tiers e.g Limit to use spells tiers like from 1-xx times per day for each Tier separately.

**Comment by FanofOld** [comment]
Great proposals,and i think this would help with some problems you can find in a game.

**Comment by Bob4001** [comment]
Yes, armor is what has always been missing in this game, also such a slot and a new skill for wearing armor could be useful
Terrain type bonuses are also an interesting idea
As for spells, the perfect way to balance them, but you also have to remember about the balance of the entire mage ruler so that he cannot cast every spell twice per turn, but maybe 2 - 3 spells more?
